src/nl/codefox/contextvh/tygronRequests.mod2g

```diff
@@ -8,6 +8,13 @@ module tygronRequests {
 		not(request(_,_,RequestID,_,_,_,_,_,_,_)))		
 		do popup_answer(RequestID,AnswerID). 
 	
+	% if a request is a confirmation that you are buying land 
+	forall percept(requests(List)),
+		bel(W\="NO_PRICE",member(request("INTERACTION","BUY_LAND",_,_,[2|T],_,W,MultiPolygon,_),List),not(requestedLand(MultiPolygon,_))) do insert(requestedLand(MultiPolygon, 1)).
+	forall percept(requests(List)),
+		bel(W\="NO_PRICE",member(request("INTERACTION","BUY_LAND",_,_,[2|T],_,W,MultiPolygon,_),List),requestedLand(MultiPolygon,Counter),Counter<3,NewCounter is Counter+1) 
+		do delete(requestedLand(MultiPolygon, Counter), doneBuying(MultiPolygon)) + insert(requestedLand(MultiPolygon, NewCounter)) + adopt(doneBuying(MultiPolygon)).
```

Comment:
Remove space between MultiPolygon and Counter for consistent coding style. Also remove space between MultiPolygon and NewCounter please.